src/main/java/frc/robot/turret/commands/TurretCommands.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot.turret.commands;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import frc.robot.RobotContainer;
+import frc.robot.turret.TurretState;
+import frc.robot.turret.TurretSubsystem;
+import frc.utils.joysticks.SmartJoystick;
+
+public abstract class TurretCommands {
```

Comment:
This is not an abstract class. I get that you don't won't to be able to create an instance of this class, but a `static` class will give the same thing, while not having a confusing usecase

Reply:
![Uploading image.png…]()

Reply:
it's giving an error.

Reply:
what do you suggest to do now?

Reply:
To read what the error says, and resolve the issue. If you need google it.

Reply:
> what do you suggest to do now?

i dont think he meant for you to put a static in the class header.
i think he meant you should just not make it abstract